CMakeLists.txt

```diff
@@ -37,11 +37,13 @@ if(NOT GZ_UTILS_VENDOR_CLI11)
   gz_find_package(CLI11 REQUIRED_BY cli PKGCONFIG_IGNORE)
 endif()
 
+gz_find_package(spdlog REQUIRED_BY log)
```

Comment:
Looks like this is generating a CMake warning on Noble.

```
-- Looking for spdlog - found

[33mCMake Warning (dev) at /usr/share/cmake/gz-cmake4/cmake4/GzFindPackage.cmake:412 (message):
   -- THIS MESSAGE IS INTENDED FOR GZ-UTILS AUTHORS --
      (IF YOU SEE THIS, PLEASE REPORT IT)

  Could not find pkg-config information for spdlog.  It was not provided by
  the find-module for the package, nor was it explicitly passed into the call
  to gz_find_package(~).  This is most likely an error in this project's use
  of gz-cmake.
Call Stack (most recent call first):
  CMakeLists.txt:40 (gz_find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.
```

Reply:
Fixed